README.md

```diff
@@ -144,3 +144,25 @@ Generates this output:
 █████████████████████████████
 ▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀
 ```
+
+## PIC generation
+
+```rust
+use qrcode::render::pic;
+use qrcode::QrCode;
+
+fn main() {
+    let code = QrCode::new(b"01234567").unwrap();
+    let image = code
+        .render()
+        .min_dimensions(1, 1)
+        .dark_color(pic::Color("#800000"))
+        .light_color(pic::Color("#ffff80"))
+        .build();
+    println!("{}", image);
+}
+```
+
+Generates PIC output that renders as follows:
+
+[![Output](examples/encode_pic.png)](examples/encode_pic.png)
```

Comment:
You should link to the text output of the PIC code, instead of the rendered PNG.

Reply:
It would be possible to have all the call to the `p(…)` macro on a single line separated by `;`, would you prefer that?

Reply:
> It would be possible to have all the call to the `p(…)` macro on a single line separated by `;`, would you prefer that?

if it is all in one line, there is no improvement in terms of readability or file size, so no.